backend/connector-integration/src/connectors/payload/transformers.rs

```diff
@@ -0,0 +1,825 @@
+                            .to_owned()
+                            .parse_value("PayloadAuth")
+                            .change_context(errors::ConnectorError::InvalidDataFormat {
+                                field_name: "auth_key_map",
+                            })?;
+                        Ok((*currency, auth))
+                    })
+                    .collect::<Result<_, Self::Error>>()?;
+                Ok(Self { auths })
+            }
+            _ => Err(errors::ConnectorError::FailedToObtainAuthType.into()),
+        }
+    }
+}
+
+// Helper function to build card request data
+fn build_payload_cards_request_data<T: PaymentMethodDataTypes>(
+    payment_method_data: &PaymentMethodData<T>,
+    connector_auth_type: &ConnectorAuthType,
+    currency: enums::Currency,
+    amount: StringMajorUnit,
+    resource_common_data: &PaymentFlowData,
+    capture_method: Option<enums::CaptureMethod>,
+    is_mandate: bool,
+) -> Result<requests::PayloadCardsRequestData<T>, Error> {
+    if let PaymentMethodData::Card(req_card) = payment_method_data {
+        let payload_auth = PayloadAuth::try_from((connector_auth_type, currency))?;
+
+        let card = requests::PayloadCard {
+            number: req_card.card_number.clone(),
+            expiry: get_card_expiry(req_card)?,
+            cvc: req_card.card_cvc.clone(),
+        };
+
+        // Get billing address to access zip and state
+        let billing_addr = resource_common_data.get_billing_address()?;
+
+        let billing_address = requests::BillingAddress {
+            city: resource_common_data.get_billing_city()?,
+            country: resource_common_data.get_billing_country()?,
+            postal_code: billing_addr.zip.clone().ok_or(
+                errors::ConnectorError::MissingRequiredField {
+                    field_name: "billing.address.zip",
+                },
+            )?,
+            state_province: billing_addr.state.clone().ok_or(
+                errors::ConnectorError::MissingRequiredField {
+                    field_name: "billing.address.state",
+                },
+            )?,
+            street_address: resource_common_data.get_billing_line1()?,
+        };
+
+        // For manual capture, set status to "authorized"
+        let status = if is_manual_capture(capture_method) {
+            Some(responses::PayloadPaymentStatus::Authorized)
+        } else {
+            None
+        };
+
+        Ok(requests::PayloadCardsRequestData {
+            amount,
+            card,
+            transaction_types: requests::TransactionTypes::Payment,
+            payment_method_type: "card".to_string(),
+            status,
+            billing_address,
+            processing_id: payload_auth.processing_account_id,
+            keep_active: is_mandate,
+        })
+    } else {
+        Err(errors::ConnectorError::NotImplemented(
+            "Payment method not implemented for Payload".to_string(),
+        )
```

Comment:
If these payment methods are not supported by payload then can we use `ConnectorError::NotSupported` error instead?